ruled_labels/specs_polkadot.yaml

```diff
@@ -0,0 +1,250 @@
+---
+name: paritytech/polkadot
+description: Import from paritytech/polkadot
+labels:
+  - name: A0-please_review
+    description: Pull request needs code review.
+    color: "666666"
+  - name: A1-needs_burnin
+    description: Pull request needs to be tested on a live validator node before merge. DevOps is notified via matrix
+    color: c7def8
+  - name: A2-insubstantial
+    description: "Pull request requires no code review (e.g., a sub-repository hash update)."
+    color: C6C7F8
+  - name: A3-in_progress
+    description: Pull request is in progress. No review needed at this stage.
+    color: DBC6F8
+  - name: A4-companion
+    description: A PR that should be considered alongside another (usually more comprehensive and detailed) PR.
+    color: EEF8C0
+  - name: A5-stale
+    description: Pull request did not receive any updates in a long time. No review needed at this stage. Close it.
+    color: F2C5F8
+  - name: A6-backport
+    description: Pull request is already reviewed well in another branch.
+    color: F8C5E7
+  - name: B0-silent
+    description: Changes should not be mentioned in any release notes
+    color: f9f0f4
+  - name: B1-note_worthy
+    description: Changes should be noted in the release notes
+    color: f9f0f4
+  - name: C1-low
+    description: PR touches the given topic and has a low impact on builders.
+    color: eeeeee
+  - name: C3-medium
+    description: PR touches the given topic and has a medium impact on builders.
+    color: eebb00
+  - name: C5-high
+    description: PR touches the given topic and has a high impact on builders.
+    color: D4C5F9
+  - name: C7-critical
+    description: PR touches the given topic and has a critical impact on builders.
+    color: E99695
+  - name: D1-audited 👍
+    description: PR contains changes to fund-managing logic that has been properly reviewed and externally audited.
```

Comment:
```suggestion
    description: PR contains changes to critical logic that has been properly reviewed and externally audited.
```

Reply:
`D*` labels will all be replaced (in a second stage once substrate also welcomed the new labels)